libs/ledgerjs/packages/errors/src/index.ts

```diff
@@ -20,6 +20,15 @@ export const AccountNotSupported = createCustomErrorClass(
   "AccountNotSupported"
 );
 export const AmountRequired = createCustomErrorClass("AmountRequired");
+export const InvalidMinimumAmount = createCustomErrorClass(
+  "InvalidMinimumAmount"
+);
+export const MayBlockAccount = createCustomErrorClass(
+  "MayBlockAccount"
+);
+export const CasperInvalidTransferId = createCustomErrorClass(
+  "CasperInvalidTransferId"
+);
```

Comment:
Since these errors are specific to Casper, they must be defined in a `libs/ledger-live-common/src/families/casper/errors.ts` file, that must get exported from `libs/ledger-live-common/src/errors.ts`

Reply:
Fixed